cmd/performance-profile-creator/cmd/root.go

```diff
@@ -38,67 +38,23 @@ import (
 type ProfileData struct {
 	isolatedCPUs, reservedCPUs string
 	nodeSelector               *metav1.LabelSelector
+	performanceProfileName     string
 }
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "performance-profile-creator",
 	Short: "A tool that automates creation of Performance Profiles",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		mcpName := cmd.Flag("mcp-name").Value.String()
-		mustGatherDirPath := cmd.Flag("must-gather-dir-path").Value.String()
-		mcp, err := profilecreator.GetMCP(mustGatherDirPath, mcpName)
+		flagData, err := getDataFromFlags(cmd)
```

Comment:
I'd rename like `profileCreatorArgsFromFlags`
Additionally: we can return a value (not a pointer) here, can't we?